homeassistant/components/simplisafe/services.yaml

```diff
@@ -1,16 +1,4 @@
 # Describes the format for available SimpliSafe services
-clear_notifications:
-  name: Clear notifications
-  description: Clear any active SimpliSafe notifications
-  fields:
-    device_id:
-      name: System
-      description: The system to remove the PIN from
-      required: true
-      selector:
-        device:
-          integration: simplisafe
-          model: alarm_control_panel
```

Comment:
Since the service is not being removed yet (just deprecated), should this be kept in for now?

Reply:
With the deprecation, we want to start driving the user towards using the button. Although it's fine to continue calling the service, we don't want to encourage new users via the UI.

See https://github.com/home-assistant/core/pull/75028#pullrequestreview-1036364800 (same situation).

Reply:
👍🏻